test/infrastructure/docker/exp/internal/controllers/dockermachinepool_controller.go

```diff
@@ -124,6 +125,10 @@ func (r *DockerMachinePoolReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	log = log.WithValues("Cluster", klog.KObj(cluster))
 	ctx = ctrl.LoggerInto(ctx, log)
 
+	if isPaused, conditionChanged, err := paused.EnsurePausedCondition(ctx, r.Client, cluster, machinePool); err != nil || isPaused || conditionChanged {
```

Comment:
```suggestion
	if isPaused, conditionChanged, err := paused.EnsurePausedCondition(ctx, r.Client, cluster, dockerMachinePool); err != nil || isPaused || conditionChanged {
```

Reply:
dropped dockerMachinePool and added it to the umbrella issue. It lacks the v1beta2 status struct.